protocol/0076-DANO-data-node.md

```diff
@@ -168,3 +167,13 @@ It must be possible to add to the data node APIs that return the result of calcu
 
 ### Schema compatibility:
 1. It is possible to identify if schema versions are NOT backwards compatible. Pull existing network snapshots start network, run a protocol upgrade to at later version and ensure both the core state and data-node data is correct (<a name="0076-COSMICELEVATOR-013" href="#0076-COSMICELEVATOR-013">0076-COSMICELEVATOR-013</a>)
+
+### General Acceptance
+* The DataNode must be able to handle brief network outages and disconnects (<a name="0076-DANO-015" href="#0076-DANO-015">0076-DANO-015</a>) 
+* All API functionality currently in the Core node must be available from the DataNode (<a name="0076-DANO-016" href="#0076-DANO-016">0076-DANO-016</a>) 
+* The validator node will only accept requests for event bus subscriptions. All other API requests will be invalid. (<a name="0076-DANO-017" href="#0076-DANO-017">0076-DANO-017</a>)  
+* The event bus stream is only available from the stream node and not the validator or DataNode (<a name="0076-DANO-018" href="#0076-DANO-018">0076-DANO-018</a>)  
```

Comment:
I think this is incorrect, validator node can also produce the event stream.

Reply:
@ze97286 - can you also confirm. Looks to me that this needs re-wording/removing